components/bl-parallax-component/src/index.js

```diff
@@ -0,0 +1,29 @@
+import { Parallax } from './react-parallax.min';
```

Comment:
I assume we could avoid using libraries like this, 400 lines of code, and most of them are typings/empty lines/comments/destructing is not so big to try to implement it yourself.

In any case, we must wait for Pods, to make parallax content dynamic.